webpack/components/Layout.jsx

```diff
@@ -13,14 +13,15 @@ export default class Layout extends React.Component {
   render() {
     return (
       <div className="page-wrap">
-        <Nav />
```

Comment:
This will be addressed in the pull request for #10. Can it be removed from this pull request pretty please?